src/api/app/models/event_subscription/form.rb

```diff
@@ -40,5 +40,14 @@ def find_or_initialize_subscription(eventtype, receiver_role, channel)
 
       EventSubscription.find_or_initialize_by(opts)
     end
+
+    def show_create_report_event?(event_class:, subscriber:)
+      if event_class.name == 'Event::CreateReport'
+        return false unless subscriber.is_moderator?
```

Comment:
Sounds an awful lot like `ReportPolicy.show`

Reply:
hmm, I think its slightly different. I thought we only sending notifications to stuff and admin for now? The `show` policy also authorizes user that created the report. But since I need the same logic at multiple places I would create another policy for that